api/GlobalEventHandlers.json

```diff
@@ -4821,7 +4821,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": true
+              "version_added": false
```

Comment:
I tested http://mdn-bcd-collector.appspot.com/tests/api/Window/onwheel and the other http://mdn-bcd-collector.appspot.com/tests/api/*/onwheel tests in IE11 and all return false, so it seems like the original data was wrong. The event itself might be supported though.